weave-js/src/components/Panel2/ChildPanelExportReport/SelectedExistingReport.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {Button} from '../../Button';
+import {Icon} from '../../Icon';
+import {ReportOption} from './utils';
+
+type SelectedExistingReportProps = {
+  selectedReport: ReportOption;
+  setSelectedReport: (report: ReportOption | null) => void;
+};
+
+export const SelectedExistingReport = ({
+  selectedReport,
+  setSelectedReport,
+}: SelectedExistingReportProps) => {
+  return (
+    <div className="mb-16 flex rounded bg-moon-50 p-8 text-moon-800">
+      <Icon name="report" className="shrink-0 grow-0 pt-4" />
+      <div className="flex grow items-center justify-between">
+        <p className="mx-8 flex  grow flex-col items-baseline gap-4">
+          <span className="text-left">{selectedReport.name}</span>
+          <span className="text-sm text-moon-500">
+            {selectedReport.projectName}
+          </span>
+        </p>
+        <Button
```

Comment:
**suggestion (non-blocking, accessibility):** icon-only buttons have no text content to explain their purpose, so it's generally a good idea to add an `aria-label`!

Reply:
Ahhh added!